contrib/devtools/github-merge.sh

```diff
@@ -83,12 +83,13 @@ function cleanup() {
 
 # Create unsigned merge commit.
 (
-  echo "Merge pull request #$PULL"
+  PRTITLE=`curl -s https://api.github.com/repos/$REPO/pulls/$PULL | grep -e '  "title": ".*",'| awk -F'"' '{print $4}'`
```

Comment:
Right, forgot we already had that defined. Although I still can't get the script to run. Only get "ERROR: Creating merge failed (already merged?)."